### PR TITLE
Game logic

### PR DESCRIPTION
**Pull request by imran5644**
Tic Tac Toe

Two players Game
The game is played on a grid that's 3 squares by 3 squares
Each player has a "X" or "O" Token
once a player get 3 marks on a row its the winner
If all 9 cells are occupied but none of the players got 3 marks in a row is a draft
i have created two classes one player and other tictactoe 
in tictactoe class all the logic related to board is there
in player all player related logic is there

### CODE REVIEW

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
# Changes Requested :recycle: 
Hello @imran5644 ,
There is an issue with your project as it seems you forgot to add all the needed files to your project. Please check my comment below and update your project accordingly.
When you are done request another review and also please always write descriptive PR messages.
slack(@ebeagusamuel)

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+# rubocop:disable Metrics/BlockLength
+require_relative '../lib/game'
+game = Tictactoe.new
+loop do
+  system('clear')
```
Comment (ebeagusamuel) [inline bin/main.rb]: - [x] Your game is trying to load a file that does not exist. Please take a look at your project again and include all the files needed to run your game.

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
# Changes Requested :recycle: 
Hello @imran5644 
It seems there is an issue with your project. Please check my comment below and update your project accordingly.
Also always write descriptive PR messages in all your PR
Do request another review when you are done.
slack(@ebeagusamuel)

File: bin/main.rb (hunk 1 above)
Comment (imran5644) [reply bin/main.rb]: i have added all the needed files. thank you for the feedback

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
Hello @imran5644 
Nice work on this project.
Your game and its flow work fine and users are able to successfully play the game.
However, there are some requirements your project needs to meet before it can be approved.
- [x] Your PR has to have a descriptive PR message. Talk a little about the game and how you've built it in the PR description.
- [x] For this milestone, you need to use at least two classes to handle the logic of the game. I would advise you split the game class you currently have into 2, maybe a player class and a board class, such that the player class handles all logic that directly deals with the player and the board class handles all logic that deals with h board.

**Review by krys2fa — APPROVED** [review]
Congratulations @imran5644! 🎊 🎉 🎈 
# Status: Project Approved! 🥇 
 You implemented all the requested changes and this milestone is approved. 👏  

- [ ] [OPTIONAL] Find below an example of a summary for a PR. You can read more on this as well as writing git commit messages in the imperative form [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/).
Screen Shot 2020-11-21 at 4 20 49 AM

You can now merge this branch into the development branch and go on with the next milestone.

💡  Do not forget to submit your project completion form after merging this branch.

All the best @imran5644!

**Pull request merged**